src/target/stm32f4.c

```diff
@@ -242,6 +245,47 @@ bool stm32f4_probe(target *t)
 	return false;
 }
 
+static bool gd32f4_attach(target *t)
+{
+	if (!cortexm_attach(t))
+		return false;
+
+	target_mem_map_free(t);
+	target_add_ram(t, 0x10000000, 0x10000); /* 64 k CCM Ram*/
+	target_add_ram(t, 0x20000000, 0x50000);     /* 320 k RAM */
+
+	/* TODO implement DBS mode */
+	const int split = 12;
```

Comment:
This isn't really an int - it can't be negative, please use a more appropriate type such as `uint8_t`. This will also bring things in line with what we've done on the fix/clang-format-cleanup branch for this file.